src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/utils.ts

```diff
@@ -554,7 +555,13 @@ export const getInitialSwedishHouseValues = (
     numberOfBathrooms: details.numberOfBathrooms,
     yearOfConstruction: details.yearOfConstruction,
     isSubleted: details.isSubleted,
-    extraBuildings: details.extraBuildings,
+    extraBuildings: details.extraBuildings.map<ExtraBuildingInput>(
```

Comment:
`ExtraBuildingInput` is not an array so shouldn't it be 
```suggestion
    extraBuildings: details.extraBuildings.map<ExtraBuildingInput[]>(
```

Reply:
Or am I stupid and this types the items in the array?

Reply:
And not the actual array?

Reply:
Not according to TS 🤷‍♂️ It screams if I add it as an array

Reply:
No I'm just confused about generics haha 😄

Reply:
Ahh yeah, map always returns an array so you just type it with the item type :P but that was confusing to me too